seqr/views/utils/pedigree_info_utils.py

```diff
@@ -49,6 +51,8 @@ def parse_pedigree_table(parsed_file, filename, user=None, project=None):
         is_datstat_upload = 'DATSTAT' in header_string
         is_merged_pedigree_sample_manifest = "do not modify" in header_string.lower() and "Broad" in header_string
         if is_merged_pedigree_sample_manifest:
+            if not user_is_pm(user):
+                raise ValueError('Unsupported file format')
```

Comment:
because this is a special broad-specific file format that only PMs are allowed to upload. If someone else tried to upload it we treat it as if its an invalid file format, same as if it was a malformed header. Realistically, this would only happen if someone tried to upload the wrong file, its not a format anyone who isn't a PM would even have access to